src/explorer/FolderTreeItem.ts

```diff
@@ -34,7 +34,7 @@ export class FolderTreeItem implements IAzureParentTreeItem {
         const httpResponse: kuduIncomingMessage = <kuduIncomingMessage>(await kuduClient.vfs.getItemWithHttpOperationResponse(this.folderPath)).response;
         // response contains a body with a JSON parseable string
         const fileList: kuduFile[] = <kuduFile[]>JSON.parse(httpResponse.body);
-        const home: string = 'home/';
+        const home: string = this.client.kind.includes('linux') ? 'home/' : 'home\\';

```

Comment:
Just as an alternative: what if you make this just 'home' without the path separator and add `+ 1` to the length below? Then you don't have to check `this.client.kind`. That being said, I can't think of an example where your existing code wouldn't work so I don't care too much.

Reply:
💡 That seems like a smarter idea.